site-src/reference/examples.md

```diff
@@ -65,7 +65,7 @@ spec:
               app: kube-dns
 ```
 
-### Story 3: Explicitly Delegate traffic to existing K8s Network Policy
+### Sample Spec for Story 3: Explicitly Delegate traffic to existing K8s Network Policy
```

Comment:
not entirely related to this specific PR but, the examples here seem a bit wrong to me :dagger: 
the `pub-svc-delegate-example` in its egress peers section is using both `namespaces` and `pods` at the same time which is not allowed? https://github.com/kubernetes-sigs/network-policy-api/blob/7321d8509d9bf425c9c5b0338db3c0fb2816b4c5/apis/v1alpha1/shared_types.go#L102
If a user uses that it will fail, Perhaps we should fix them or open an issue and I can copy some from the conformance test yamls which are tested working samples (I've been running them on my cluster) into the website.

Reply:
Good catch this is wrong 

It should just be `pods` with that namespace selector and podSelector

@Dyanngg Can you fixup the examples here?

Reply:
Fixed